testrunner/runner/BUILD

```diff
@@ -12,14 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-load(
-    "@io_bazel_rules_go//go:def.bzl",
-    "go_binary",
-)
-load(
-    "@io_bazel_rules_docker//go:image.bzl",
-    "go_image",
-)
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
 
 binary_srcs = [
```

Comment:
rm these?

Reply:
Removing 'load' line breaks the build. Is this some other suggestion?

Committing as is.